core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -95,7 +95,8 @@ class LoadBalancerService(
     info(this, s"blackboxFraction = $blackboxFraction")
 
     /** We run this often on an invoker before going onto the next. */
-    private val activationCountBeforeNextInvoker = 10
+    private val activationCountBeforeNextInvoker = Math.max(1, config.loadbalancerActivationCountBeforeNextInvoker.toInt)
```

Comment:
@rabbah 
I can extract the property as a numeric value, the same way it is done for the property controllerBlackboxFraction :

val controllerBlackboxFraction = this.getAsDouble(WhiskConfig.controllerBlackboxFraction, 0.10)

I've already tested this way on my local machine.

The good thing with this approach is, that parsing errors come up at startup time, and not when the property is used at some point in runtime.